python/nav/junos/nav_views.yml

```diff
@@ -96,6 +96,14 @@ ElsEthernetSwitchingInterfaceTable:
     key: l2iff-interface-handle
     view: ElsEthernetSwitchingInterfaceView
 
+ElsEthernetSwitchingInterfaceTableJunOS20:
+    rpc: get-ethernet-switching-interface-details
+    args:
+        detail: True
+    item: l2ng-l2ald-iff-interface-entry
+    key: l2iff-interface-handle
+    view: ElsEthernetSwitchingInterfaceView
+
```

Comment:
I would be tempted to preface this block with "JunOS is being precious and special again" :)